client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -7,48 +7,123 @@ import CardMedia from '@material-ui/core/CardMedia';
 import Typography from '@material-ui/core/Typography';
 import DeleteOutlineIcon from '@material-ui/icons/DeleteOutline';
 import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import { useUser } from '../../context/useUserContext';
 
 import useStyles from './useStyles';
+import uploadImagesAPI from '../../helpers/APICalls/uploadImages';
+
+interface UploadImagesState {
+  background: string | File;
+  profile: string | File;
+}
 
 const ProfilePhoto = (): JSX.Element => {
-  const [imgSrc, setImgSrc] = useState<string>('');
+  const [uploadImages, setUploadImages] = useState<UploadImagesState>({
+    background: '',
+    profile: '',
+  });
+  const { userState, dispatchUserContext } = useUser();
   const classes = useStyles();
   const { loggedInUser } = useAuth();
-
-  const handleImageUpload = (event: React.ChangeEvent<HTMLInputElement>) => {
-    if (event.target.files?.length) {
-      const src = URL.createObjectURL(event.target.files[0]);
-      setImgSrc(src);
+  const { updateSnackBarMessage } = useSnackBar();
+  const handleImageUpload = (event: ChangeEvent<HTMLInputElement>): void => {
+    if (userState.background === '' && event.target.files?.length) {
+      const background = URL.createObjectURL(event.target.files[0]);
+      dispatchUserContext({ type: 'UPLOAD_BACKGROUND', background });
+      // eslint-disable-next-line

```

Comment:
what is currently the eslint issue here?

Reply:
`Forbidden non-null assertion  @typescript-eslint/no-non-null-assertion`

Reply:
This is resolved when I  store `event.target.files` in a variable.